### PR TITLE
Manhattan plot tool

### PR DESCRIPTION
**Pull request by astrovsky01**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Review by davebx — APPROVED** [review]

File: tools/novoplasty/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: novoplasty
```
Comment (mvdbeek) [inline tools/novoplasty/.shed.yml:1]: Was this added intentionally ?

File: tools/qq_tools/qq_manhattan.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<tool id='qq_manhattan' name='Manhattan Plots' version='0.1.0' python_template_version='3.5'>
```
Comment (mvdbeek) [inline tools/qq_tools/qq_manhattan.xml:1]:
```suggestion
<tool id="qq_manhattan" name="Manhattan Plots" version="0.1.0" profile="19.05">
```
Can you please replace the single quotes within xml elements with double quotes?

File: tools/qq_tools/qq_manhattan.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <requirements>
+        <requirement type='package' version='3.6'>r-base</requirement>
```
Comment (mvdbeek) [inline tools/qq_tools/qq_manhattan.xml:3]: r-base will be installed with any of the other packages

File: tools/qq_tools/qq_manhattan.xml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        <requirement type='package' version='0.1.4'>r-qqman</requirement>
+        <requirement type='package' version='1.6.4'>r-optparse</requirement>
+        <requirement type='package' version='1.0.2'>r-dplyr</requirement>
+        <requirement type='package' version='1.3.0'>r-tidyverse</requirement>
```
Comment (mvdbeek) [inline tools/qq_tools/qq_manhattan.xml:7]:
```suggestion
```

It doesn't look like your using these ?

File: tools/novoplasty/.shed.yml (hunk 1 above)
Comment (astrovsky01) [reply tools/novoplasty/.shed.yml:1]: Yes -- I rebased too far back and can't figure out how to undo...

Comment (astrovsky01) [reply tools/novoplasty/.shed.yml]: Ok, fixed the files, if not the actual commit log

File: tools/qq_tools/qq_manhattan.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<tool id="qq_manhattan" name="Manhattan Plots" version="0.1.0" python_template_version="3.5">
```
Comment (mvdbeek) [inline tools/qq_tools/qq_manhattan.xml:1]:
```suggestion
<tool id="qq_manhattan" name="Manhattan Plots" version="0.1.0" profile="19.05">
```

File: tools/qq_tools/.shed.yml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    - Visualization
+    - Variant Analysis
+homepage_url: https://CRAN.R-project.org/package=qqman
+auto_tool_repositories:
+    name_template: "{{ tool_id }}"
+    description_template: "Manattan plot tool from the qqman package"
+suite:
+    name: suite_qq
+    description: "Tools from the qq package"
+    type: repository_suite_definition
```
Comment (mvdbeek) [inline tools/qq_tools/.shed.yml:11]: Is it likely that there will be other tools ? Otherwise I'd just drop the suite key

Comment (astrovsky01) [reply tools/qq_tools/.shed.yml:11]: It's possible, and I wanted to leave that open in case anybody else made any

**Comment by mvdbeek** [comment]
Thanks @astrovsky01!

**Pull request merged**